### PR TITLE
In progress

### PR DESCRIPTION
**Pull request by mtmgoliath**
Hopefully not toooooooo much code to parse through! 

Thanks =]

### CODE REVIEW

**Review by wintersanchez — APPROVED** [review]

**Review by wintersanchez — CHANGES_REQUESTED** [review]
add vendor prefixes and media queries to make the mobile version work better!

**Review by wintersanchez — CHANGES_REQUESTED** [review]
fix the media query thing

File: style.css [hunk 1]
```diff
@@ -75,15 +143,49 @@ h1 {
     padding: 5rem 2rem;
 }
 
+.portfolioHighlight {
```
Comment (wintersanchez) [inline style.css:149]: put 1rem on left and right instead of just 2rem on left so that the images stay in the center

File: style.css [hunk 2]
```diff
@@ -92,18 +194,54 @@ footer {
...
+
+    .portfolioBlurb {
+        font-size: 1.5rem;
+        margin: 0rem 3rem;
+    }
+}
+
+@media screen and (max-width: 500px) {
+
+    .jumbotron {
```
Comment (wintersanchez) [inline style.css:220]: fix this media query so that the content doesn't get cut off on the left side of the screen and overflow.

File: style.css [hunk 3]
```diff
@@ -24,12 +22,37 @@ header {
...
+}
+
+.navLinks {
+    display: flex;
+    justify-content: center;
+    text-align: center;
+    font-size: 2rem;
+}
+
+.navLinks a {
```
Comment (wintersanchez) [inline style.css:42]: scroll issue here too on mobile media query. overflow situation!

**Review by wintersanchez — CHANGES_REQUESTED** [review]
fix the issues in the comments please thanks

File: style.css (hunk 1 above)
Comment (mtmgoliath) [reply style.css:149]: done =]

File: style.css [hunk 4]
```diff
@@ -39,51 +62,133 @@ header {
...
+    margin-left: 4rem;
 }
 
-h1 {
+h2 {
     color: #ffffff;
     background-color: #000000;
 }
 
 .jumbotron p {
```
Comment (wintersanchez) [inline style.css:86]: make it so that the text has the same vertical space between each line and so that it all stays above the button element

**Review by wintersanchez — CHANGES_REQUESTED** [review]

File: style.css (hunk 4 above)
Comment (wintersanchez) [reply style.css:86]: perhaps just check this again as i think this still needs fixing -- i tried removing the margin: left part and that made the text stay above the button

**Pull request merged**